README.md

```diff
@@ -324,6 +324,12 @@ Ensures that the element is visible to a user. Currently, the following checks a
 - Element has a size (its [bounding box](https://developer.mozilla.org/en-US/docs/Web/API/Element/getBoundingClientRect) has a non-zero width and height)
 - Element's opacity is greater than 0.05 (opacity of parent elements are considered)
 
+#### Target size
+
+Ensures that the clickable region of an element is large enough to easily be activated by users, even users with reduced fine motor skills or while using a touch input device.
```

Comment:
@calebeby How would you feel quoting this first paragraph so we don't have to reinvent the wheel?

> https://www.w3.org/WAI/WCAG21/Understanding/target-size.html

Source: https://www.w3.org/WAI/WCAG21/Understanding/target-size.html

Reply:
We could even link to that resource also.